post_proc.py

```diff
@@ -1,8 +1,48 @@
-# Pra remover os espaços em branco no início, lstrip
-
 in_file = open("diario-anadia-2022-08-29-extraido.txt", "r")
-out_text = in_file.read().lstrip()
-out_file = open("diario-anadia-2022-08-29-proc.txt", "w")
-out_file.write(out_text)
-out_file.close()
+#Mudei o nome da variável out_text para in_text, pois ela não será mais usada como texto de saída.
+in_text = in_file.read().lstrip()
+in_text = in_text.splitlines()
+
+#Função que retorna o índice da primeira palavra do preâmbulo
+def find_initial_id():
+    initial_id = 0
+    for index, line in enumerate(in_text):
```

Comment:
Como as linhas são contíguas, por que não marcar o início e o fim do preâmbulo. Daí já pode remover esses elementos da lista.